### PR TITLE
Bug 472154 - [j2x] Copyright header and javadoc is sometimes missing in converted class

### PR DESCRIPTION
**Pull request by dhuebner**

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=472154

**Comment by dhuebner** [comment]
Jenkins xtend-runtime-tests-master-latest

File: plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/javaconverter/JavaASTFlattener.xtend [hunk 1]
```diff
@@ -401,12 +401,23 @@ class JavaASTFlattener extends ASTVisitor {
...
+				assignedComments.add(it)
+			]
+		}
 		decreaseIndent
 		appendLineWrapToBuffer
 		appendToBuffer("}")
 		return false
 	}
 
+	def private Iterable<Comment> unAssignedComments(CompilationUnit cu) {
```
Comment (szarnekow) [inline plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/javaconverter/JavaASTFlattener.xtend]: this should probably be called unassignedComments (lowercase a) or getUnassignedComments

**Comment by dhuebner** [comment]
Jenkins xtend-runtime-tests-maintenance-latest

File: plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/javaconverter/JavaASTFlattener.xtend (hunk 1 above)
Comment (dhuebner) [inline plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/javaconverter/JavaASTFlattener.xtend]: Would like to rename after merging into maintenance. my local branch is in a bad state

**Pull request merged**